<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="23" hostname="DESKTOP-DJ4FDME" failures="5" tests="5" name="Test(failed)" time="0.181" errors="0" timestamp="2024-11-08T20:12:31 IST">
  <testcase classname="demoTestNG.TestScenario4" name="second_testcase" time="0.039">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario4.second_testcase(TestScenario4.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="seventh_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario6.seventh_testcase(TestScenario6.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- seventh_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="forth_testcase" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario8.forth_testcase(TestScenario8.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="first_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello] but found [Hello world]">
      <![CDATA[java.lang.AssertionError: expected [Hello] but found [Hello world]
at demoTestNG.TestScenario.first_testcase(TestScenario.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="second_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demoTestNG.TestScenario.second_testcase(TestScenario.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="seventh_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- seventh_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="seventh_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- seventh_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="demoTestNG.TestScenario4" name="seventh_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- seventh_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="demoTestNG.TestScenario6" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario8" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="demoTestNG.TestScenario" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
</testsuite> <!-- Test(failed) -->
